Tools/c-analyzer/c_analyzer/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 
 def analyze(filenmes, **kwargs):
-    results = iter_analyis_results(filenames, **kwargs)
+    results = iter_analysis_results(filenames, **kwargs)
```

Comment:
@ericsnowcurrently Potentially impactful change?

Reply:
Hmm, clearly this function isn't getting used anywhere. 😞  I'm fine with the fix.